os/hal/src/hal_sdc.c

```diff
@@ -690,19 +693,13 @@ bool sdcConnect(SDCDriver *sdcp) {
     goto failed;
   }
 
-  /* Switches to high speed.*/
-  if (HAL_SUCCESS != detect_bus_clk(sdcp, &clk)) {
-    goto failed;
-  }
-  sdc_lld_set_data_clk(sdcp, clk);
-
   /* Reads extended CSD if needed and possible.*/
   if (SDC_MODE_CARDTYPE_MMC == (sdcp->cardmode & SDC_MODE_CARDTYPE_MASK)) {
 
     /* The card is a MMC, checking if it is a large device.*/
     if (_mmcsd_get_slice(sdcp->csd, MMCSD_CSD_MMC_CSD_STRUCTURE_SLICE) > 1U) {
       uint8_t *ext_csd = sdcp->buf;
-
+      osalThreadSleepMilliseconds(10);
```

Comment:
The newer driver code does these sleeps in a slightly different way - do you think we need to copy?

Reply:
can you share an example.